packages/cohere/src/cohere-embedding-model.ts

```diff
@@ -86,7 +86,7 @@ export class CohereEmbeddingModel implements EmbeddingModelV2<string> {
     return {
       embeddings: response.embeddings.float,
       usage: { tokens: response.meta.billed_units.input_tokens },
-      rawResponse: { headers: responseHeaders },
+      rawResponse: { headers: responseHeaders, body: response },
```

Comment:
needs to be `rawValue` from `await postJsonToApi`, see one of the language model providers for an example. `value: response` is parsed by Zod and won't include everything, `rawValue: rawResponse` would be the unparsed json (also below)